hawc/apps/summary/models.py

```diff
@@ -417,6 +418,24 @@ def get_prisma_data(cls, assessment: Assessment) -> str:
             }
         )
 
+    @classmethod
+    def get_data_from_config(cls, config):
```

Comment:
perhaps use a pydantic schema here to ensure config matches? we could create a `schema.py` if that'd be helpful.